misc/dist/windows/godot.iss

```diff
@@ -1,8 +1,8 @@
-#define MyAppName "Godot Engine"
+#define MyAppName "Blazium Engine"
 #define MyAppVersion "4.3"
-#define MyAppPublisher "Godot Engine contributors"
-#define MyAppURL "https://godotengine.org/"
-#define MyAppExeName "godot.exe"
+#define MyAppPublisher "Blazium Engine contributors"
+#define MyAppURL "https://blaziumengine.app/"
```

Comment:
The website link should be `https://blazium.app/`

Reply:
oops, thanks